src/infill.cpp

```diff
@@ -368,7 +378,7 @@ void Infill::generateConcentricInfill(VariableWidthPaths& toolpaths, const Setti
         }
 
         const coord_t inset_wall_count = iterative ? 1 : std::numeric_limits<coord_t>::max();
-        WallToolPaths wall_toolpaths(current_inset, infill_line_width, inset_wall_count, wall_0_inset, settings);
+        WallToolPaths wall_toolpaths(current_inset, line_width_to_use, inset_wall_count, wall_0_inset, settings);
```

Comment:
If I understand correctly that if there is a gap larger then `line_width_to_use` then this gap would be stitched. By making this variable larger for ironing, ironing gaps would would be stitched less often.

Is this the only use for this variable, cause if it is not then i would say this solution is overly specific for this error-case, and might introduce bugs in the other instances where this variable is used.